Examples/MAX32520/UCL/project.mk

```diff
@@ -0,0 +1,24 @@
+# This file can be used to set build configuration
+# variables.  These variables are defined in a file called 
+# "Makefile" that is located next to this one.
+
+# For instructions on how to use this system, see
+# https://github.com/Analog-Devices-MSDK/VSCode-Maxim/tree/develop#build-configuration
+
+#BOARD=MAX32520FTHR
+# ^ For example, you can uncomment this line to make the 
+# project build for the "MAX32520FTHR" board.
+
+# **********************************************************
+
+# Enable UCL library
+LIB_UCL = 1
+export UCL_VERSION=2.7.0
```

Comment:
Consider declaring this as a `LIB_UCL` variable in `libs.mk`.

Ex:  
```Makefile
...
LIB_UCL ?= 0
ifeq ($(LIB_UCL), 1)

UCL_VERSION ?= 2.7.0
...
```

Reply:
This is not a manual process, the version parse from ucl_xxx.a file and this version could be different device to device, 
In future for one device it can be 2.8.0 but for another one it can stay as 2.7.0....